icepyx/__init__.py

```diff
@@ -1,3 +1,3 @@
 from icepyx.core.query import Query
 
-from _version import version as __version__
+from icepyx.core._version import version as __version__
```

Comment:
```suggestion
from _version import version as __version__
```
@tsutterley, maybe you need to revert the change here?

Reply:
Ah yes, I see what you mean. But I think this problem should be fixed once we merge #175 (though I'll double check to be sure). See also https://github.com/icesat2py/icepyx/pull/172#discussion_r577778898 (that PR was merged into #175).

Reply:
Sounds good.  I'll revert these changes and let them be handled in #175

Reply:
#175 has been merged into 'development'. I've pulled in the changes locally into this `rgt` branch and `pip install` seems to work, i.e.:

```
git switch rgt
git merge development
conda activate icepyx
pip install -e .
python -c "import icepyx as ipx; print(ipx.__version__)"
# 0.3.2b1.dev17+g336779c
```
Could you try and see if it works on your side?

Reply:
yep! the changes in #175 work (also doesn't have to be installed in "editable" mode so the changes should work for people installing with pypi once they are released)